src/pages/starter-kit/integration/create-integration.md

```diff
@@ -200,12 +204,12 @@ Use the following steps to download and configure the Adobe Commerce integration
 
 When configuring the `COMMERCE_BASE_URL` environment variable, the format differs between [PaaS and SaaS](#paas-or-saas):
 
-   For PaaS (On-Premise/Cloud):
+&#8203;<Edition name="paas" />For PaaS (On-Premise/Cloud):
```

Comment:
Is this kind of labelling too duplicative, since we are calling out via the tag and via text?

Reply:
Maybe a little, but I don't think it's a show stopper. We can always refine later.